src/components/VideoDisplay/VideoCard/VideoCard.jsx

```diff
@@ -44,15 +45,17 @@ export const VideoCard = ({
     }
   };
 
-  const unlikeHandler = (_id) => {
+  const unlikeHandler = (e, _id) => {
+    e.stopPropagation();
     unlikeVideo(_id, token, likesDispatch, setLiked);
   };
 
   const isLiked = (_id) => {
     likes.find((video) => video._id === _id) ? setLiked(true) : setLiked(false);
   };
```

Comment:
You can probably avoid state here if you can use array,some() and let the value get decided at run time.